Languages/IronPython/IronPythonCompiler/Program.cs

```diff
@@ -272,17 +272,27 @@ class Program {
             var compileOptions = new Dictionary<string, object>() {
                 { "mainModule", config.MainName }
             };
-
-            ClrModule.CompileModules(DefaultContext.DefaultCLS, 
-                Path.ChangeExtension(config.Output, ".dll"), 
-                compileOptions, 
-                config.Files.ToArray());
-
-            if (config.Target != PEFileKinds.Dll) {
-                GenerateExe(config);
+            
+            try
+            {
+                ClrModule.CompileModules(DefaultContext.DefaultCLS, 
+                    Path.ChangeExtension(config.Output, ".dll"), 
+                    compileOptions, 
+                    config.Files.ToArray());
+                
+                var outputfilename = Path.ChangeExtension(config.Output, ".dll");
+                if (config.Target != PEFileKinds.Dll) {
+                    outputfilename = Path.ChangeExtension(config.Output, ".exe");                    
+                    GenerateExe(config);
+                }
+                ConsoleOps.Info("Saved to {0}", config.Output);
```

Comment:
Yes, sorry, and I thought I carefully checked it line by line after I tested it :( which I had to do separately as the 2.7.7 source code doesn't compile under VS2013 which is what I use - though 2.7.5 did as I recall.